src/txn/txn_rollback_to_stable.c

```diff
@@ -114,14 +116,18 @@ __rollback_abort_update(WT_SESSION_IMPL *session, WT_ITEM *key, WT_UPDATE *first
          * of the rollback to stable page read, it instantiates the tombstones on the page.
          * The transaction id validation is ignored in all scenarios except recovery.
          */
-        if (!__rollback_txn_visible_id(session, upd->txnid) ||
-          rollback_timestamp < upd->durable_ts || upd->prepare_state == WT_PREPARE_INPROGRESS) {
+        txn_id_visible = __rollback_txn_visible_id(session, upd->txnid);
+        if (!txn_id_visible || rollback_timestamp < upd->durable_ts ||
+          upd->prepare_state == WT_PREPARE_INPROGRESS) {
             __wt_verbose_multi(session, WT_VERB_RECOVERY_RTS(session),
               "rollback to stable update aborted with txnid: %" PRIu64
-              " durable timestamp: %s and stable timestamp: %s, prepared: %s",
-              upd->txnid, __wt_timestamp_to_string(upd->durable_ts, ts_string[0]),
+              ", txnid not visible: %s, stable timestamp (%s) < durable timestamp (%s): %s, "
+              "prepared: %s",
+              upd->txnid, !txn_id_visible ? "true" : "false",
+              __wt_timestamp_to_string(upd->durable_ts, ts_string[0]),
```

Comment:
I think this line needs to be below the rollback_timestamp line. The print format has 'stable timestamp' first then 'durable timestamp' and these calls have `upd->durable_ts` then `rollback_timestamp`.